### DISCUSSION TITLE
首页最上方的个性化推荐视频区域能移植到动态页面右侧么

### DISCUSSION THREAD

**Discussion by u2shana** [the1812/Bilibili-Evolved#1850] [功能建议 / Ideas]
不知道有没有其他用户和我一样，只看动态页面，
觉得首页内容太杂乱区域太多，加载也慢。

B站web端没出这功能之前就有人搞出了出来，不过我觉得这功能放在动态页面没新动态的时候刷下更适合
https://greasyfork.org/zh-CN/scripts/368446-bilibili%E7%BD%91%E9%A1%B5%E7%AB%AF%E6%B7%BB%E5%8A%A0app%E9%A6%96%E9%A1%B5%E6%8E%A8%E8%8D%90

**Comment by Geeyun-JY3** [comment]
~~https://github.com/the1812/Bilibili-Evolved/issues/1203#issuecomment-727571895~~

~~估计放到动态首页也不会做~~

见 https://github.com/the1812/Bilibili-Evolved/discussions/1850#discussioncomment-733998

**Comment by the1812** [comment]
以后有机会可以把这功能带上
~~我平时也直接进动态的~~
